ivy/functional/ivy/statistical.py

```diff
@@ -487,6 +487,81 @@ def var(
         containing the variance; otherwise, a non-zero-dimensional array containing the
         variances. The returned array must have the same data type as x.
 
+    This function conforms to the `Array API Standard
+    <https://data-apis.org/array-api/latest/>`_. This docstring is an extension of the
+    `docstring <https://data-apis.org/array-api/latest/API_specification/generated/signatures.elementwise_functions.tan.html>`_
+    in the standard.
+
+    Both the description and the type hints above assumes an array input for simplicity,
+    but this function is *nestable*, and therefore also accepts :code:`ivy.Container`
+    instances in place of any of the arguments.
+
+    Functional Examples
+    -------------------
+
+    With :code:`ivy.Array` input:
+
+    >>> x = ivy.array([0, 1, 2, 3, 4, 5, 6, 7, 8])
+    >>> y = ivy.var(x)
+    >>> print(y)
+    ivy.array(6.66666667)
+
+    >>> x = ivy.array([[0,1,2], [3,4,5],[6,7,8]])
+    >>> y = ivy.var(x)
+    >>> print(y)
+    ivy.array(6.66666667)
+
+    >>> x = ivy.array([[0,1,2], [3,4,5],[6,7,8]])
+    >>> y = ivy.var(x, axis=1)
+    >>> print(y)
+    ivy.array([0.667, 0.667, 0.667])
+
+    >>> x = ivy.array([[0,1,2], [3,4,5],[6,7,8]])
+    >>> y = ivy.var(x, axis=0)
+    >>> print(y)
+    ivy.array([6., 6., 6.])
+
```

Comment:
Please add more functional examples according to [docstring-examples-guide](https://lets-unify.ai/ivy/deep_dive/12_docstring_examples.html#docstring-examples) here.

Reply:
I missed some points. Updated.

Reply:
Add docstring and docstring examples to `var` in `ivy/array/statistical.py` as well as in `ivy/container/statistical.py`

Reply:
Make sure to request review once you have made the changes.

Reply:
Updated, please help and review.